packages/wrangler/src/__tests__/sentry.test.ts

```diff
@@ -0,0 +1,133 @@
+import * as fs from "node:fs";
+import * as fsp from "node:fs/promises";
+import os from "node:os";
+import path from "node:path";
+import * as TOML from "@iarna/toml";
+import * as Sentry from "@sentry/node";
+import prompts from "prompts";
+
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { runInTempDir } from "./helpers/run-in-tmp";
+import { runWrangler } from "./helpers/run-wrangler";
+const { reportError } = jest.requireActual("../reporting");
+
+describe("Error Reporting", () => {
+  runInTempDir({ homedir: "./home" });
+  mockConsoleMethods();
+
+  const originalTTY = process.stdout.isTTY;
```

Comment:
Can you add one more test that checks that Sentry is not called if `isTTY` is false?

Reply:
Done. 👍🏻 
https://github.com/cloudflare/wrangler2/pull/227/files#diff-530006b680b300438ce683119071a6c83caeff8ddd876b9f886efb7e172e24b7R134-R153